### PR TITLE
Docs/link fixing

### PR DESCRIPTION
**Pull request by JordanL8**
### Purpose of this PR
Fixed links, replaced references to Lightweight, updated the ReadMe, updated ShaderGraph Nodes.

### CODE REVIEW

**Comment by GrantLamb-Unity** [comment]
We should address this issue in this PR:
[Shader Graph] Package Description makes no mention of support for Universal RP - https://fogbugz.unity3d.com/f/cases/1173068/

I've confirmed that this change isn't in the branch.

**Comment by JordanL8** [comment]
I'll fix that now, thanks for pointing it out.

**Comment by JordanL8** [comment]
@GrantLamb-Unity fixed now.

File: README.md [hunk 1]
```diff
@@ -76,31 +75,31 @@ You can create multiple Pipeline Assets to store settings for different built pl
 
 To create a Render Pipeline Asset: 
 
-1. In the Project window, navigate to a directory outside of the Scriptable Render Pipeline Folder, then right click in the Project window and select ___Create > Render Pipeline >  High Definition or Lightweight > Render Pipeline/Pipeline Asset.___
+1. In the Project window, navigate to a directory outside of the Scriptable Render Pipeline Folder, then right click in the Project window and select ___Create > Render Pipeline > Rendering > High Definition or Universal Render Pipeline Asset.___
```
Comment (Verasl) [inline README.md]: This is incorrect for Universal RP
image

**Pull request merged**